tests/cnf/core/network/dpdk/tests/rootless.go

```diff
@@ -652,7 +652,7 @@ func defineAndCreateVlanNad(name, intName string, vlanID uint16, ipam *nad.IPAM)
 func defineAndCreateSrIovNetwork(srIovNetwork, resName string, vlanID uint16) {
 	srIovNetworkObject := sriov.NewNetworkBuilder(
 		APIClient, srIovNetwork, NetConfig.SriovOperatorNamespace, tsparams.TestNamespaceName, resName).
-		WithMacAddressSupport()
+		WithMacAddressSupport().WithLogLevel("debug")
```

Comment:
Do we want to create a variable for "debug"?

Reply:
changed